packages/cli-kit/src/git.test.ts

```diff
@@ -84,3 +85,21 @@ describe('ensurePresentOrAbort', () => {
     await expect(ensurePresentOrAbort()).resolves.toBeUndefined()
   })
 })
+
+describe('ensureInsideGitDirectory', () => {
```

Comment:
This one is a great candidate to be an integration test in a temporary directory. I'd recommend creating a `git.integration.test.ts` module and move this piece there as an integration test that does:

1. I create a temporary directory
2. I call `ensureInsideGitDirectory`
3. I expect an error